framework/auth/core.py

```diff
@@ -99,9 +99,10 @@ def validate_social(value):
 
 
 def validate_email(item):
```

Comment:
As discussed, it would be good to adapt from more complete email validators such as Django's implementation: https://github.com/django/django/blob/master/django/core/validators.py#L137